src/w2utils.js

```diff
@@ -247,7 +248,7 @@ var w2utils = (function ($) {
     }
 
     function isDateTime (val, format, retDate) {
-        if (format == null) format = w2utils.settings.dateFormat + '|' + w2utils.settings.timeFormat;
+        if (format == null) format = w2utils.settings.datetimeFormat;
```

Comment:
Sure, but it is default format. There is always a way to define custom for any particular case. Anyway, I think it is not a big issue. The pull request is merged. Thank you for your contribution.